source/_templates/installations/elastic/common/copy_certificates_filebeat.rst

```diff
@@ -1,6 +1,6 @@
 .. Copyright (C) 2020 Wazuh, Inc.
 
-In section **Installing Elasticsearch**, a .zip that contains the necessary files was created. The zip file must be copied from the host where was created into the Wazuh server host. ``scp`` can be used for example. This example assumes that the file has been copied into the home folder.
+In section **Installing Elasticsearch**, the ``cert.zip`` file was created. The file must be copied into the Wazuh server host, for example, with a use of ``scp``. This guide assumes that the file is placed in ~/ (home user folder).
```

Comment:
It should be "with an use of", nevertheless, I think that in this context "use" is uncountable, so maybe you could use "for example, using ``scp``".